app/src/main/java/com/glia/exampleapp/MainFragment.java

```diff
@@ -72,6 +73,8 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                 navigateToCall(GliaWidgets.MEDIA_TYPE_VIDEO));
         view.findViewById(R.id.message_center_activity_button).setOnClickListener(v ->
                 navigateToMessageCenter());
+        view.findViewById(R.id.screen_share_test_button).setOnClickListener( v ->
+                navigateToScreenShare());
```

Comment:
It's good that you've added the "Screen share test button". 👍 However, considering that in one of the next tickets we should open the new "End screen sharing" screen from a Floating Action Button, I think there is no need to add a test button to our testing app. So, I would rather remove the code related to test button as it should still be removed in one of the upcoming tickets. Does it make sense to you?

Reply:
True, this was purely for testing purposes. MOB-1825 actually can be tested through this logic, should I remove it now or after that PR is reviewed?

Reply:
I think it's better to remove it. Usually, I'm adding such a king of functionality for testing only locally, and do not commit/push it. So, to me - it's better to remove it.